dhcp-controller/controllers/dhcpserver_controller.go

```diff
@@ -143,18 +167,23 @@ func (r *DHCPServerReconciler) genConfigMap(dhcpserver dhcpv1alpha1.DHCPServer)
 
 func (r *DHCPServerReconciler) ensureServer(request reconcile.Request,
 	server dhcpv1alpha1.DHCPServer,
-	dep *appsv1.Deployment,
```

Comment:
No, it will not change here. There will be one instance of DHCPServer VMPool which will make a single replica VM . This will keep that VM up at all times